app/components/gc-textarea.js

```diff
@@ -1,23 +1,35 @@
 import Ember from "ember";
 import AutoResizableTextarea from "./auto-resize-textarea";
-import AjaxPromise from "stock/utils/ajax-promise";
-const { getOwner } = Ember;
 
 export default AutoResizableTextarea.extend({
   previousValue: "",
   store: Ember.inject.service(),
   item: null,
 
   didInsertElement() {
-    const value = (this.get("value") || "").trim();
-    if (!value && this.get("require")) {
+    this.handleSiblings();
+  },
+
+  handleSiblings() {
+    if (this.hasError()) {
       return this.showSiblings();
     }
     this.hideSiblings();
   },
 
+  hasError() {
+    const value = (this.get("value") || "").trim();
+    if (!value && this.get("require")) {
+      return true;
+    }
+    return false;
+  },
```

Comment:
```suggestion
  hasError() {
    const value = (this.get("value") || "").trim();
    return !value && this.get("require");
  },
```

(not really a change request, just pointing it out)